<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>main_window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0">
    <item>
     <widget class="QChartView" name="plot" native="true"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Data Sources</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPV_1">
        <property name="text">
         <string>libera1:bpm1:tdp_synth_long.X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPV_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPV_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPV_4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbSuppressDC">
        <property name="text">
         <string>Suppress DC ?</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Sampling Frequency (MHz)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtFs"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Number of Points</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtN">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Post-Processing</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbScale">
          <item>
           <property name="text">
            <string>Normalize by N</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Power Spectrum</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Update Sources</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbEnable">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>794</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">QChartView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
